infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplier.java

```diff
@@ -194,6 +196,7 @@ private void addSidecar(
             .build();
 
     InternalMachineConfig machineConfig = machineResolver.resolve();
+    machineConfig.getAttributes().put(CONTAINER_TYPE_ATTRIBUTE, "tool-container");
```

Comment:
It would be better to make value a constant too

Reply:
Done.